pytition/petition/templates/petition/petition_detail.html

```diff
@@ -51,7 +51,7 @@
 
 {% block extracss %}
 <link href="{% static "css/petition.css" %}" rel="stylesheet" type="text/css">
-{% if petition.has_share_buttons %}
+{% if petition.has_email_share_button or petition.has_facebook_share_button or petition.has_tumblr_share_button or petition.has_linkedin_share_button or petition.has_twitter_share_button or petition.has_mastodon_share_button or petition.has_whatsapp_share_button %}
```

Comment:
you could add a `has_any_share_button()` method in the `Petition` class that does this boolean computation, just for the sake of keeping the template easy to read.